Cargo.toml

```diff
@@ -157,3 +160,6 @@ tracing-opentelemetry = "0.17"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 uuid = { version = "1", features = ["serde", "v4"] }
 wiremock = "0.5"
+
+[patch.crates-io]
+reqsign = { git = "https://github.com/messense/reqsign.git", branch = "ureq-disable-default-feature" }
```

Comment:
Let's wait for reqsign.

Reply:
reqsign v0.6.4 has been released: https://github.com/Xuanwo/reqsign/discussions/173

Thanks @messense